src/SprykerEco/Zed/Econda/Business/Exporter/AbstractExporter.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Exporter;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\FailedResultTransfer;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\WriterInterface;
+
+abstract class AbstractExporter implements ExporterInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Econda\Dependency\Plugin\EcondaPluginInterface[]
+     */
+    protected $collectorPlugins = [];
+
+    /**
+     * @var \Generated\Shared\Transfer\FailedResultTransfer
+     */
+    protected $failedResultTransfer;
+
+    /**
+     * @var \Generated\Shared\Transfer\BatchResultTransfer
+     */
+    protected $batchResultTransfer;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Business\Exporter\Writer\WriterInterface
+     */
+    protected $writer;
+
+    /**
+     * @var \Spryker\Zed\Touch\Persistence\TouchQueryContainerInterface
+     */
+    protected $queryContainer;
+
+    /**
+     * AbstractExporter constructor.
+     *
```

Comment:
Additional comments for `constructor` are not needed — only parameter definitions